cli/ssh.go

```diff
@@ -72,6 +73,11 @@ func ssh() *cobra.Command {
 				return err
 			}
 
+			updateWorkspaceBanner, outdated := verifyWorkspaceOutdated(client, workspace)
+			if outdated && isTTYOut(cmd) {
```

Comment:
Ah, this tty check is only for `stdout` which doesn't help us (we want to confirm stderr.)

Made the following quick test:

```
❯ go run main.go >/dev/null
stdout isatty: false
stderr isatty: true

❯ go run main.go 2>/dev/null
stdout isatty: true
stderr isatty: false
```

Using:

```go
package main

import (
	"fmt"
	"io"
	"os"

	"github.com/mattn/go-isatty"
)

func main() {
	both := io.MultiWriter(os.Stdout, os.Stderr)
	fmt.Fprintf(both, "stdout isatty: %v\n", isatty.IsTerminal(os.Stdout.Fd()))
	fmt.Fprintf(both, "stderr isatty: %v\n", isatty.IsTerminal(os.Stderr.Fd()))
}
```

Arguably, this is a bug we potentially have across our whole CLI, but for now I think we can just add a new function: `isTTYErr`.